dataset_eval/gaia_wrapper.py

```diff
@@ -44,9 +46,29 @@ def train_model(project_dir, groundtruth_file, filelist_file):
         datasets_dir=os.path.join(project_dir, "datasets"),
         results_dir=os.path.join(project_dir, "results"),
     )
+    """
+    Converting the preferences from string type to list
+    """
+    c_value = [int(value) for value in c_value.split(',')]
+    gamma_value = [int(s) for s in gamma_value.split(',')]
+    preprocessing_values = [str(value) for value in preprocessing_values.split()]
+    """
+    Updating C, Gamma, and preprocessing values in the project file which were
+    taken as input by the user
+    """
+    update_parameters(project_file, c_value, gamma_value, preprocessing_values)
     run_tests(project_file)
     return select_best_model(project_dir)
 
+def update_parameters(project_file, c_value, gamma_value, preprocessing_values):
```

Comment:
We should have a docstring explaining what this function does and what values its arguments take. See https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings